paasta_tools/kubernetes_tools.py

```diff
@@ -1865,13 +1868,90 @@ def update_stateful_set(
     )
 
 
+def get_events_for_object(
```

Comment:
Seems like this is the same thing as `list_namespaced_event()`?  https://github.com/kubernetes-client/python/blob/master/kubernetes/docs/CoreV1Api.md#list_namespaced_event